r/NEWS.md

```diff
@@ -19,6 +19,37 @@
 
 # arrow 13.0.0.9000
 
+## Breaking changes
+
+## New features
+
+* Can't read in partitioning column in CSV datasets when both (non-hive) partition and schema supplied (#37658)
```

Comment:
A bit torn on this, as this felt more like a bugfix than a new feature, though happy to leave it in here from a messaging point-of-view (I guess technically we'd never implemented it in the first place...)